PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcalEmbeddingHelper.h

```diff
@@ -358,6 +360,10 @@ class AliAnalysisTaskEmcalEmbeddingHelper : public AliAnalysisTaskSE {
   std::vector <std::string>                     fFilenames        ; ///<  Paths to the files to embed
   std::string                                   fConfigurationPath; ///<  Path to %YAML configuration
   std::vector <std::string>                     fEmbeddedRunlist  ; ///<  Good runlist for files to embed
+  std::vector <std::string>                     fEmbeddedRunblock ; ///<  Good runblock for files to embed, each run corresponds to the first run of a block
+  Int_t                                         fEmbeddedRunblockMin; ///< Minimum run in the run range valid for embedding
+  Int_t                                         fEmbeddedRunblockMax; ///< Maximum run in the run range valid for embedding, this one is first of next block  
+  Int_t                                         fDataRunNumber    ;   ///< Current run number in data file  
```

Comment:
If you remove files that you're not interested in, it's not clear to me if even this variables are necessary. But perhaps I'm being too simple for this

Reply:
This is just in case of future development when the run data changes and in case a mistake is done to know when the run number changed